generators/client/templates/angular/src/main/webapp/app/shared/alert/_alert-error.component.ts

```diff
@@ -77,7 +79,7 @@ export class <%=jhiPrefixCapitalized%>AlertErrorComponent implements OnDestroy {
                                 fieldError.objectName + '.' + convertedField)<% } else { %>convertedField.charAt(0).toUpperCase() +
                                 convertedField.slice(1)<% } %>;
                             this.addErrorAlert(
-                                'Field ' + fieldName + ' cannot be empty', 'error.' + fieldError.message, { fieldName });
+                                'Error on field "' + fieldName + '"', 'error.' + fieldError.message, { fieldName });
```

Comment:
All those texts in the alert error component are in the Typescript (by default), and passed to the addAlertError function.
As it's used as a directive, I don't know exactly where do you want to put this default text. With some inputs in the directive ?